src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/stores/DatagridStore.js

```diff
@@ -44,6 +44,10 @@ export default class DatagridStore {
         this.setPage(1);
         this.loadingStrategy = loadingStrategy;
 
+        if (this.localeInterceptDisposer) {
+            this.localeInterceptDisposer();
```

Comment:
Erfan renamed this variable in #3638, so after this is merged, I have to rebase this PR.